Ports/c-ares/package.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='c-ares'
+version='1.19.0'
+files="https://c-ares.org/download/c-ares-${version}.tar.gz c-ares-${version}.tar.gz bfceba37e23fd531293829002cac0401ef49a6dc55923f7f92236585b7ad1dd3"
+auth_type='sha256'
+useconfigure='true'
+configopts=("-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt")
```

Comment:
While it's nice that the new IPv6 defintions make ares happier, we should pass whatever  "disable IPv6" configure option exists for the library in configopts here. Trying to use the mostly-unimplemented IPv6 support in the serenity kernel will only lead to pain. Patches welcome to make IPv6 actually work :)

Reply:
So unfortunately I dont think there is a build option to disable IPv6. I looked through bits of their source code and especially the paths where the IPv6 definitions we had to add are used, but there are no points where anything leading to the macros I had to define can be optionally build or not. Just to be safe I also tried building with disable-ipv6 and similar options.

Reply:
Ah, that's unfortunate. I confirmed there's no such option by looking at the CMakeLists.txt upstream.